crates/valence_schem/examples/schem_loading.rs

```diff
@@ -0,0 +1,83 @@
+use std::path::PathBuf;
+
+use clap::Parser;
+use valence::bevy_app::AppExit;
+use valence::client::despawn_disconnected_clients;
+use valence::client::event::default_event_handler;
+use valence::prelude::*;
+use valence_schem::Schematic;
+
+const SPAWN_POS: BlockPos = BlockPos::new(0, 256, 0);
+
+#[derive(Parser)]
+#[clap(author, version, about)]
+struct Cli {
+    /// The path to a Sponge Schematic.
+    path: PathBuf,
+}
+
+pub fn main() {
+    tracing_subscriber::fmt().init();
+
+    App::new()
+        .add_plugin(ServerPlugin::new(()))
+        .add_system_to_stage(EventLoop, default_event_handler)
+        .add_system_set(PlayerList::default_system_set())
+        .add_startup_system(setup)
+        .add_system(init_clients)
+        .add_system(despawn_disconnected_clients)
+        .run();
+}
+
+fn setup(world: &mut World) {
+    let Cli { path } = Cli::parse();
+
+    if !path.exists() {
+        eprintln!("File `{}` does not exist. Exiting.", path.display());
+        world.send_event(AppExit);
+    } else if !path.is_file() {
+        eprintln!("`{}` is not a file. Exiting.", path.display());
+        world.send_event(AppExit);
+    }
+
+    let mut instance = world
+        .resource::<Server>()
+        .new_instance(DimensionId::default());
+
+    match Schematic::load(path) {
+        Ok(schem) => {
+            schem.paste(&mut instance, SPAWN_POS, |_| BiomeId::default());
+        }
+        Err(err) => {
+            eprintln!("Error loading schematic: {err}");
+            world.send_event(AppExit);
+        }
+    }
```

Comment:
It would be nice to include this kind of thing in doc comments so it's visible to people looking at the docs.

Reply:
What exactly do you think should be in doc comments?

Reply:
A short explanation of what schematics are, examples for basic usage with a little context. You can look at the doc comments for the inventory module for reference.